tools/gh-pages-publish.ts

```diff
@@ -0,0 +1,31 @@
+const { cd, exec, echo, touch } = require("shelljs")
```

Comment:
Are we planning to use gh-pages for anything?

Reply:
no, I deleted.